keyboards/acheron/elongate/delta/delta.c

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2021 Gondolindrim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "delta.h"
+
+#define LED_PIN_ON_STATE 1
+// Inits all indicator LEDs as push-pull outputs
+void led_init_ports(void) {
+    palSetLineMode(LED1_PIN, PAL_MODE_OUTPUT_PUSHPULL);
+    palSetLineMode(LED2_PIN, PAL_MODE_OUTPUT_PUSHPULL);
+    palSetLineMode(LED3_PIN, PAL_MODE_OUTPUT_PUSHPULL);
+    palSetLineMode(LED4_PIN, PAL_MODE_OUTPUT_PUSHPULL);
+    palSetLineMode(LED5_PIN, PAL_MODE_OUTPUT_PUSHPULL);
+    palSetLineMode(LED6_PIN, PAL_MODE_OUTPUT_PUSHPULL);
+}
+
+// This function updates LEDs 1, 2 and 3 according to num, caps and scroll lock states
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(LED1_PIN, !led_state.num_lock);
+        writePin(LED2_PIN, !led_state.caps_lock);
+        writePin(LED3_PIN, !led_state.scroll_lock);
+    }
+    return res;
+}
+
+// Turns off all bottom LEDs
+void turn_off_bottom_leds(void){
+	writePin(LED3_PIN, 1);
+	writePin(LED4_PIN, 1);
+	writePin(LED5_PIN, 1);
```

Comment:
This looks like it should be LEDs 4-6 instead of 3-5.

```suggestion
	writePin(LED4_PIN, 1);
	writePin(LED5_PIN, 1);
	writePin(LED6_PIN, 1);
```

Reply:
Indeed it should! Committed in bfe1c583d8